sessions/forecast-evaluation.qmd

```diff
@@ -98,13 +99,17 @@ And raises challenges about doing statistically rigorous "testing" when trying t
 We will now generate some forecasts from one of the models we used earlier and evaluate them.
 However, to set us up for a more multi-dimensional experiment later in this session, we will start by loading in a bit more data than we did before.
```

Comment:
I'm realizing that the data in the evaluation session is for a few more weeks than the data in the concepts session. Maybe we should load in the maximum number of weeks needed for any session and then we can subset it down in other sessions?